src/api/general-services/pipeline-manage/index.js

```diff
@@ -296,7 +296,7 @@ const createPipeline = async (experimentId, processingConfigUpdates) => {
   // appropriate.
   // eslint-disable-next-line consistent-return
   const mergedProcessingConfig = _.cloneDeepWith(processingConfig, (o) => {
-    if (_.isObject(o) && !o.dataIntegration && typeof o.enabled === 'boolean') {
+    if (_.isObject(o) && !o.dataIntegration && !o.embeddingSettings && typeof o.enabled === 'boolean') {
```

Comment:
This means that it still won't work with enabled right? 
(Enabled has nevertheless been removed from data ingest and dynamo)

Reply:
No, it will just simply ignore the whole 'per-sample' nonsense for the object (if an object does not have it, it gets parsed, i.e. it never gets parsed if the object has `embeddingSettings` as a parameter).